distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Time_Of_Day.enso

```diff
@@ -284,17 +284,6 @@ type Time_Of_Day
             duration : Duration -> self.minus_builtin duration
             _ : Period -> Error.throw (Time_Error.Error "Time_Of_Day does not support date intervals (periods)")
 
-    ## Format this time of day as text using the default formatter.
-
-       > Example
-         Convert the current time to text.
-
-             from Standard.Base import Time_Of_Day
-
-             example_to_text = Time_Of_Day.now.to_text
-    to_text : Text
-    to_text self = @Builtin_Method "Time_Of_Day.to_text"
```

Comment:
Other date/time types do not have this.

Having it breaks the `to_text` invocation on type.

I.e. `Time_Of_Day.to_text` now works. With that shadow definition it wasn't working properly.

It's a bit of a workaround to #1538